lib/dice_bag/template_helpers.rb

```diff
@@ -17,5 +17,56 @@ def ensure_is_private_key(key)
         raise "The private key provided is invalid"
       end
     end
+
+    def generate_509_certificate(private_key, root_ca: nil, root_key: nil)
+      root_key ||= OpenSSL::PKey::RSA.new 2048 # the CA's public/private key
+      root_ca ||= default_root_ca(root_key)
```

Comment:
`||=` has no effect since those are local vars

Reply:
yes it does, it will only assign if the var is nil, I prefer this pattern than putting a ruby call as a default parameter value

Reply:
ah sorry I didn't see the method signature 👍